### DISCUSSION TITLE
Improve prEditedNotification message notification

### DISCUSSION THREAD

**Discussion by ikarami** [renovatebot/renovate#24178] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.57.2

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps Server

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

`prEditedNotification` message mentions a checkbox which might not (or does not) exist in PR description when renovate PRs are created. It is misleading for end-users when they try to understand if something is broken.

See this line where the problematic message is defined in a constant:
https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/update/branch/handle-existing.ts#L64

Because the problem is already located to the line mentioned above, I am not preparing a reproducible sample as I don't think it is needed.

Note: this message can be suppressed, but I think it still holds some value and we want to have it posted on our PRs. It's just about wording which should be improved in order not to mislead users.

Note 2: ideal solution would include a possibility to specify your own messages, but I understand that it would be much more work than simply modifying one constant.

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
PRs welcome

**Comment by ikarami** [comment]
Thanks, please take a look at it here:
https://github.com/renovatebot/renovate/pull/24200
